api/src/main/java/org/openmrs/module/fhir2/api/util/XmlPatchUtils.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.fhir2.api.util;
+
+import static ca.uhn.fhir.util.StringUtil.toUtf8String;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+
+import ca.uhn.fhir.context.FhirContext;
+import ca.uhn.fhir.rest.api.Constants;
+import ca.uhn.fhir.rest.server.exceptions.InvalidRequestException;
+import com.github.dnault.xmlpatch.Patcher;
+import org.hl7.fhir.instance.model.api.IBaseResource;
+
+public class XmlPatchUtils {
+	
+	/**
+	 * Handles xml patch operations ("application/xml-patch+xml")
```

Comment:
Could you put some Javadocs in here? E.g.,

```suggestion
	 * Applies an XML Patch to a FHIR Resource
	 *
	 * @param
```

Reply:
done!